src/components/button/button.jsx

```diff
@@ -24,13 +29,9 @@ module.exports = React.createClass({
   },
 
   renderAnchor: function() {
-    var props = {
-      id: this.props.id,
-      className: classNames('component-button', this.props.size, this.props.purpose),
-      href: this.props.href,
-      target: this.props.target,
-      onClick: this.props.handleClick
-    };
+    const props = _.omit(this.props, 'children');
```

Comment:
should this be `let` instead of `const`? we're technically mutating the object on the next line